### PR TITLE
Remove datacat module from Richard Clamp

### PR DESCRIPTION
**Pull request by phaedriel**
#### Pull Request (PR) description

Remove datacat module from Richard Clamp

#### This Pull Request (PR) fixes the following issues

- https://github.com/voxpupuli/puppet-elasticsearch/issues/980

#### Pull request acceptance prerequisites:

- [x] Signed the [CLA](https://www.elastic.co/contributor-agreement/) (if not already signed)
- [x] Rebased/up-to-date with base branch
- [x] Tests pass
- [ ] Updated CHANGELOG.md with patch notes (if necessary)
- [ ] Updated CONTRIBUTORS (if attribution is requested)

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -156,25 +156,21 @@
...
     # Generate Elasticsearch config
-    $_es_config = merge(
+    $data = merge(
       $elasticsearch::config,
       { 'path.data' => $elasticsearch::datadir },
       { 'path.logs' => $elasticsearch::logdir },
       $_tls_config
     )
 
-    datacat_fragment { 'main_config':
```
Comment (bastelfreak) [inline manifests/config.pp:166]: any idea why this was used in the first place? To concatenate multiple strings into one file?

Comment (phaedriel) [reply manifests/config.pp:166]: I suppose it was to merge 1 or more data in one file (with older puppet version)

**Review by smortex — APPROVED** [review]
Looks like a good move.

**Comment by phaedriel** [comment]
Could i do something more ?

**Comment by smortex** [comment]
> Could i do something more ?

The heads up is fine: I wanted to let the opportunity for another review / somebody to raise concerns before this get merged and without further GitHub notification I lost it.

**Pull request merged**